doc/source/tune/examples/ml-frameworks.rst

```diff
@@ -108,3 +108,11 @@ At the end of these guides you will often find links to even more examples.
         .. button-ref:: tune_train_torch_example
 
             End-to-end Example for Tuning a PyTorch Model with PBT
+
+    .. grid-item-card::
```

Comment:
The latest versions of the Ray Tune example docs have moved away from the "item card" and [seems to have reorganized and moved towards using text links instead.](https://docs.ray.io/en/latest/tune/examples/#id1) I'll run your example tomorrow when I have time, but in the meantime is it possible for you to pull the latest changes from master and update the example with a text link instead of the image? :) Thank you!